docs/contributing/setup.md

```diff
@@ -2,117 +2,15 @@
 
 ## Setup
 
-You will need to install these tools on your machine:
+Refer to the specific instructions for each platform:
 
-### macOS
+ - [macOS](./setup-macos.md)
+ - [Windows](./setup-windows.md)
+ - [Linux](./setup-linux.md)
 
-See [mac-deps-setup.md](./setup-macos.md).
+Experimental support for building Desktop is also available for these platforms:
 
-### Windows
-
- - [Node.js v8.12.0](https://nodejs.org/dist/v8.12.0/)
-    - *Make sure you allow the Node.js installer to add node to the PATH.*
- - [Python 2.7](https://www.python.org/downloads/windows/)
-    - *Let Python install into the default suggested path (`c:\Python27`), otherwise you'll have
-      to configure node-gyp manually with the path which is annoying.*
-    - *Ensure the **Add python.exe to Path** option is selected.*
- - One of Visual Studio 2015, Visual C++ Build Tools or Visual Studio 2017
-   - [Visual C++ Build Tools](http://go.microsoft.com/fwlink/?LinkId=691126)
-     - *Run `npm config set msvs_version 2015` to tell node to use this toolchain.*
-   - Visual Studio 2015
-     - *Ensure you select the **Common Tools for Visual C++ 2015** feature as that is required by Node.js
-        for installing native modules.*
-     - *Run `npm config set msvs_version 2015` to tell node to use this toolchain.*
-   - [Visual Studio 2017](https://www.visualstudio.com/vs/community/)
-     - *Ensure you select the **Desktop development with C++** feature as that is required by Node.js for
-        installing native modules.*
-     - *Run `npm config set msvs_version 2017` to tell node to use this toolchain.*
-
-### Fedora 26
-
-First, add the NodeJS package repository for 8.x.
-
-```shellsession
-$ curl --silent --location https://rpm.nodesource.com/setup_8.x | sudo bash -
-```
-
-After that, install the dependencies to build and test the app:
-
-```shellsession
-$ sudo dnf install -y nodejs gcc-c++ make libsecret-devel libXScrnSaver
-```
-
-If you want to package Desktop for distribution, you will need these additional dependencies:
```

Comment:
The packaging side for Linux this has changed so much since I wrote these docs, and none of these packages and hacks are now needed.